src/components/ApplnBar.js

```diff
@@ -123,9 +144,67 @@ const ApplnBar = () => {
       return <DynamicTabs component1={<ProcessPage />} component2={'c2'} />;
     } else if (page === 'VEHICLE') {
       return <AddVehicle />;
+    } else if (page === 'USER') {
+      return <User />;
+    } else if (page === 'HOME') {
+      return <Home />;
     }
   };
 
+  const onLoginSuccess = (res) => {
+    console.log('Login Success: currentUser:', res.profileObj);
+    setImgUrl(res.profileObj.imageUrl);
+    setAvatarVisible('visible');
+    setLoginVisible(false);
+    setLogoutVisible('visible');
+    refreshTokenSetup(res);
+    setUserName(res.profileObj.name);
+    setEmail(res.profileObj.email);
+    localStorage.setItem('authToken', res.tokenId);
+    refreshTokenSetup(res);
+  };
+
+  const onLoginFailure = (res) => {
+    console.log('Login failed: res:', res);
+    showAlert(`Failed to login`, 'error');
+  };
+
+  const onLogoutSuccess = (res) => {
+    localStorage.setItem('authToken', null);
+    setAnchorEl(null);
+    console.log('Logged out Success');
+    showAlert('Logged out Successfully');
+    setAvatarVisible('hidden');
+    setLoginVisible(true);
+    setLogoutVisible('hidden');
+    //ToDo: route to home page
+  };
+
+  const onLogoutFailure = (res) => {
+    console.log('Log out Failed');
+    alert('Log out Failed');
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  const handleClick = (event) => {
```

Comment:
There are multiple things in this component now. Please mention click of what. Same for close.